### DISCUSSION TITLE
Add padding at TOCPageBreak

### DISCUSSION THREAD

**Discussion by vijayasanthi1993** [mpdf/mpdf#1769] [Q&A]
Code: 
$this->m_pdf->pdf->WriteHTML('<tocpagebreak toc-prehtml="&lt;h6&gt;INDEX&lt;/h6&gt;&lt;p&gt;The reports and statements set out below comprise the financial statements presented to the shareholder:&lt;/p&gt;" toc-margin-bottom="50px">');

I want to add padding/margin between each title. Also i want to remove that dot,dot dot lines.kindly help me
![Screenshot (10)](https://user-images.githubusercontent.com/54358980/205436707-48739dff-6ca7-4bec-9aa3-b699417c22ad.png)

**Comment by finwe** [comment]
You can control the ToC visuals with CSS as [stated in the documentation](https://mpdf.github.io/what-else-can-i-do/table-of-contents.html). 

Dottab in autogenerated TOC cannot currently be hidden.
